run/deployment-previews/check_status.py

```diff
@@ -129,7 +129,7 @@ def github_token(project_id: str, ghtoken_secretname: str) -> str:
 
     # The secret was encoded for you as part of the secret creation, so decode it now.
     github_token = response.payload.data.decode("UTF-8")
-    return github_token
+    return github_token.strip()
```

Comment:
```py
github_token = response.payload.data.decode("UTF-8").strip()
```

probably makes more sense if we're storing the correct token in that var.

O/w LGTM.

Reply:
sgtm. thanks!